src/modules/esl-share/core.ts

```diff
@@ -1,3 +1,9 @@
+export type {ESLShareButtonTagShape} from './core/esl-share-button.shape';
+export type {ESLShareListTagShape} from './core/esl-share-list.shape';
+export type {ESLSharePopupTagShape} from './core/esl-share-popup.shape';
+export type {ESLSharePopupTriggerTagShape} from './core/esl-share-popup-trigger.shape';
+export type {ESLShareTagShape} from './core/esl-share.shape';
```

Comment:
It might be better to do that inside of exact modules.
The types are not a problem of bundle, and we expect the "good" user will use shortcuts and tree shaking. But it will be more convenient for us to link related things.
⚠️ Discussable